dataactvalidator/config/sqlrules/b11_object_class_program_activity_3.sql

```diff
@@ -3,4 +3,7 @@ SELECT
     op.object_class
 FROM object_class_program_activity AS op
 WHERE op.submission_id = {}
-AND op.object_class = '000'
+AND LENGTH(op.object_class) <> 3
+AND op.object_class <> '0'
```

Comment:
We were only looking for '000' so we want to keep looking for that along with any permutation of it. In this case, we want to use
```
AND (op.object_class = '000' OR op.object_class = '00' OR op.object_class = '0')
```
only formatted nicer